pikaur/main.py

```diff
@@ -90,12 +90,12 @@ def cli_print_upgradeable() -> None:
     if args.quiet:
         print_stdout('\n'.join([
             pkg_update.name for pkg_update in updates
-        ]))
+        ]), end='')
     else:
         print_stdout(pretty_format_upgradeable(
             updates,
             print_repo=PikaurConfig().sync.get_bool('AlwaysShowPkgOrigin')
-        ))
+        ), end='')
```

Comment:
won't it result in missing linebreak at the end of the output?

Reply:
The recent test failed because of `AssertionError: 'etc-update\n' != 'etc-update'` so I'm trying this.

Reply:
mb in that test to use splitlines, idk

Reply:
inb4 the test will fail because of...
```python
#! /usr/bin/python3
print('\n'.join(['PKG1', 'PKG2']), end='')
```
```bash
$ python3 /tmp/test.py
PKG1
PKG2%
```
```bash
$ python3 /tmp/test.py | wc -l
1
```
The lesser evil might be to use `rstrip('\n')` instead. What do you want to do next?

Reply:
I can't test counting lines. See the comment above. It should fail the counting test. All the more reason to add a dedicated counting lines test.

Which diff do you prefer? Here's another one.
```diff
diff --git a/pikaur_test/test_sysupgrade.py b/pikaur_test/test_sysupgrade.py
index 9a5fa55..1b67bfe 100644
--- a/pikaur_test/test_sysupgrade.py
+++ b/pikaur_test/test_sysupgrade.py
@@ -75,10 +75,8 @@ class SysupgradeTest(PikaurDbTestCase):
 
         self.downgrade_dev_pkg()
 
-        query_result = pikaur('-Qu').stdout
-        self.assertEqual(
-            len([l for l in query_result.splitlines() if l]), 0
-        )
+        query_result = pikaur('-Qu').stdout.rstrip('\n')
+        self.assertEqual(len(query_result.splitlines()), 0)
 
         # and finally test the sysupgrade itself
         pikaur('-Su --noconfirm --devel --ignore pikaur-git')
@@ -96,14 +94,13 @@ class SysupgradeTest(PikaurDbTestCase):
         self.downgrade_repo1_pkg()
         self.downgrade_aur1_pkg()
 
-        query_result = pikaur('-Quq --aur').stdout.strip()
+        query_result = pikaur('-Quq --aur').stdout
         self.assertEqual(
-            query_result, self.aur_pkg_name
+            query_result, f'{self.aur_pkg_name}\n'
         )
-
-        query_result = pikaur('-Quq --repo').stdout.strip()
+        query_result = pikaur('-Quq --repo').stdout
         self.assertEqual(
-            query_result, self.repo_pkg_name
+            query_result, f'{self.repo_pkg_name}\n'
         )
 
         query_result = pikaur('-Qu').stdout
```